internal/worker/worker.go

```diff
@@ -11,7 +12,10 @@ import (
 	"github.com/mauricioabreu/mosaic-video/internal/storage"
 )
 
-const LockingTimeTTL time.Duration = 120 * time.Second
+const (
+	LockingTimeTTL    time.Duration = 120 * time.Second
+	KeepAliveInterval time.Duration = 30 * time.Second
```

Comment:
`KeepAliveInterval` could be based on `Locking TimeTTL` right? Maybe 1/3 of it

Reply:
Oh right, it could be.